languages/javascript/exercises/concept/array-transformations/.docs/instructions.md

```diff
@@ -0,0 +1,73 @@
+Elyse, magician-to-be, continues her training. She will be given a deck of cards and attempt to transform that deck. She will make certain cards appear, disappear, change their values, or completely rearrange themselves.
```

Comment:
I like how the existing theme is being re-used!

Reply:
Just following the instructions. :)